zavro/decoder.go

```diff
@@ -8,8 +8,68 @@ import (
 	"github.com/brimdata/zed"
 	"github.com/brimdata/zed/zcode"
 	"github.com/go-avro/avro"
+	"github.com/riferrei/srclient"
```

Comment:
For what it's worth, I recently introduced a schema registry client in franz-go, `github.com/twmb/franz-go/pkg/sr`. I plan to keep it as a separate module, but I'm thinking to 1.0.0 the mod soon. If you'd like to provide API feedback before I stabilize, I'd appreciate it :D

Reply:
Also the below encoders / decoders could potentially be done with the `Serde` type, although I don't have single flight right now -- this may be an area for feedback.